DNA-RNA.py

```diff
@@ -0,0 +1,58 @@
+seq = str(input()).upper()
```

Comment:
It`s a good idea to check, what is your input. For example, unsuitable symbols.

Reply:
Some checks were added